Documentation/README.md

```diff
@@ -26,10 +26,9 @@
 # Building from Source
 
   * [Build System Configuration](building/configuration.md)
-  * [Build Dependencies for Linux and macOS](building/unix/dependencies.md)
-  * [Building Xamarin.Android on Linux and macOS](building/unix/instructions.md)
-  * [Build Dependencies for Windows](building/windows/dependencies.md)
-  * [Building Xamarin.Android on Windows](building/windows/instructions.md)
+  * [Build Dependencies](building/dependencies.md)
```

Comment:
No; this is reverting 494fc57ba68f2c49ab04537ca4d9d01670eb16e7, and we don't want to do that.